src/tokenized-line.coffee

```diff
@@ -321,8 +321,11 @@ class TokenizedLine
     return unless maxColumn?
     return unless @text.length > maxColumn
 
-    if /\s/.test(@text[maxColumn])
-       # search forward for the start of a word past the boundary
+    if isCjkCharacter(@text[maxColumn])
+      # always wrap when a CJK character is at the wrap boundary
+      return maxColumn
```

Comment:
Yeah, I tried to reason about always wrapping when the line contained CJK characters and the edge case you're pointing out is basically the reason why I avoided doing so. In particular, I was trying understand what we would have done by using that approach with the following javascript line of code:

```js
/* 화화화화화화화화화화화화화화화화화화화화화화화화 */ const hello = "OK"
```

If the wrap occurs in the middle of either `const` or `hello`, for instance, I don't think we'd want to break those words in two like this:

```js
/* 화화화화화화화화화화화화화화화화화화화화화화화화 */ con
st hello = "OK"
```